appveyor.yml

```diff
@@ -26,5 +31,19 @@ after_build:
 
     $wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "DesktopClrTestResults.xml"))
 
+on_success:
+- ps: >-
+    if ($env:APPVEYOR_REPO_TAG -eq "true")
+    {
+        nuget push .\build\Castle.Core.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.log4netIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.NLogIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
+        nuget push .\build\Castle.Services.Logging.SerilogIntegration.*.nupkg -ApiKey $env:NUGET_API_KEY -Source https://api.nuget.org/v3/index.json
```

Comment:
I don't mind if we leave symbols for another day, I assume we'd need to remove the wildcard here though.